hw/ip/spi_host/data/spi_host_testplan.hjson

```diff
@@ -45,7 +40,7 @@
               - Ensure transactions are transmitted/received correctly
             '''
       milestone: V2
-      tests: ["spi_host_perf"]
+      tests: []
```

Comment:
Perhaps this is a dumb question, but what does it mean if there is a test point with no tests specified?  (I feel like I should know this already...)

Reply:
just means the test that exercises this test point has not been implemented yet.
I removed the tests for two reasons
1. the build system will give you warnings for all the tests defined in the hjson but not yet implemented.
2. I don't plan on making a test for each test point. most will be covered by one or two tests.